libres/lib/res_util/res_version.cpp

```diff
@@ -0,0 +1,41 @@
+#include <ert/util/util.hpp>
+
+#include <ert/res_util/res_version.hpp>
+
+#define xstr(s) #s
+#define str(s) xstr(s)
+
+const char* res_version_get_git_commit() {
+    #ifdef GIT_COMMIT
+        return str(GIT_COMMIT);
+    #else
+        return "Unknown git commit hash";
+    #endif
+}
+
+const char* res_version_get_build_time() {
+    #ifdef COMPILE_TIME_STAMP
+        return COMPILE_TIME_STAMP;
+    #else
+        return "Unknown build time";
+    #endif
+}
+
+int res_version_get_major_version() {
+  return 99;
+}
+
+
+int res_version_get_minor_version() {
+  return 0;
+}
+
+
+const char * res_version_get_micro_version() {
+  return "0";
+}
+
+
+bool res_version_is_devel_version() {
+    return util_sscanf_int(str(RES_VERSION_MICRO), NULL);
```

Comment:
Any reason why we kept `RES_VERSION_MICRO` here?

Reply:
I have tried to minimize the change set, so any change not necessary in order to get build or test passing, have been ignored. We can certainly rip out all parts made redundant by this merge. I can start looking at what it would take.

Reply:
Reverted.